projects/plugins/social/src/js/components/admin-page/index.jsx

```diff
@@ -18,7 +18,49 @@ import React from 'react';
  */
 import styles from './styles.module.scss';
 
+const getConnectionsDiv = () => {
+	// eslint-disable-next-line no-undef
+	if ( ! connections ) {
+		return null;
+	}
+	// eslint-disable-next-line no-undef
+	const providers = Object.keys( connections );
+	return providers.map( provider => {
+		// eslint-disable-next-line no-undef
+		const connectionIds = Object.keys( connections[ provider ] );
+		const connectionDiv = connectionIds.map( connectionId => {
+			// eslint-disable-next-line no-undef
+			const connection = connections[ provider ][ connectionId ];
+			return (
+				<div>
+					<tr>
+						<th>Name</th>
+						<th>Image</th>
```

Comment:
It's fine for now, but in general you'd want to use the [i18n package](https://www.npmjs.com/package/@wordpress/i18n) for all strings like this.